android/src/main/java/com/reactnativeone/OneModule.java

```diff
@@ -206,6 +246,35 @@ private OneLogLevel logLevelFromInteger(Integer logLevel) {
     return result;
   }
 
+  // Convert OneResponse Class to HashMap so React can read it.
+  private WritableNativeMap responseObjectToReadableMap(OneResponse response) {
```

Comment:
I don't see the null guard up at the call sites. Check that the response is not null before proceeding here.

Reply:
Added